src/Legacy/Storage.php

```diff
@@ -2931,42 +2915,4 @@ protected function findContent($tablename, $contentId)
 
         return $oldContent;
     }
-
-    /**
-     * Setter for pager storage element.
-     *
-     * @param string      $name
-     * @param array|Pager $pager
-     */
-    public function setPager($name, $pager)
-    {
-        static::$pager[$name] = ($pager instanceof Pager) ? $pager : new Pager($pager, $this->app);
-
-        return $this;
-    }
-
-    /**
-     * Getter of a pager element. Pager can hold a paging snapshot map.
-     *
-     * @param string $name Optional name of a pager element. Whole pager map returns if no name given.
-     *
-     * @return array
-     */
-    public function &getPager($name = null)
-    {
-        if ($name) {
-            if (array_key_exists($name, static::$pager)) {
-                return static::$pager[$name];
-            } else {
-                return false;
-            }
-        } else {
-            return static::$pager;
-        }
-    }
-
-    public function isEmptyPager()
-    {
-        return (count(static::$pager) === 0);
-    }
```

Comment:
I  know we've ripped the head off a lot of stuff, but this class is kind of central. Probably better to leave these as empty functions so that they don't throw an exception on deprecated use.

Can we :fire: this class already :grin: